JASP-Desktop/components/JASP/Widgets/FileMenu/PrefsUI.qml

```diff
@@ -69,12 +69,42 @@ ScrollView
 					checked:			preferencesModel.currentThemeName === "darkTheme"
 					onCheckedChanged:	preferencesModel.currentThemeName  =  "darkTheme"
 					toolTip:			qsTr("Switches to a dark theme, makes JASP a lot easier on the eyes for those night owls out there.")
-					KeyNavigation.tab:	languages
-					KeyNavigation.down:	languages
+					KeyNavigation.tab:	uiFont
+					KeyNavigation.down:	uiFont
 				}
 			}
 		}
 
+		PrefsGroupRect
+		{
+			id:		fontGroup
+			title:	qsTr("Font")
+
+			TextArea
+			{
+				id:					uiFont
+				width:				300 * jaspTheme.uiScale
+				height:				150 * jaspTheme.uiScale
+				title:				qsTr("List of Font")
+				toolTip:			qsTr("Font family used by the interface")
+				text:				preferencesModel.interfaceFont.replace(/,/g, "\n")
+				onApplyRequest:
+				{
+					preferencesModel.interfaceFont = text.split("\n")
```

Comment:
Code duplication...
Could you not do that? 

Maybe move the function to C++ and use it there?
Although if you change to a combobox this will probably not be necessary anymore.